src/lib/runtime_config/runtime_config.ml

```diff
@@ -1025,6 +1025,12 @@ module Epoch_data = struct
   module Data = struct
     type t = { ledger : Ledger.t; seed : string }
     [@@deriving bin_io_unversioned, yojson]
+
+    let gen =
+      let open Quickcheck.Generator.Let_syntax in
+      let%bind ledger = Ledger.gen in
+      let%map seed = String.gen_nonempty in
```

Comment:
I don't seed is just some random seed. It should be `Epoch_seed`.

Reply:
@ghost-not-in-the-shell The last time we communicated, you mentioned looking into the generator for the Epoch_seed. Did you discover anything of use? Thanks.

Reply:
I've been told that for the epoch_seed, there's a gen function called Mina_base.Epoch_seed.gen, and we can convert the seed into a string by Mina_base.Epoch_seed.to_base58_check function.

Reply:
@Sventimir This might be helpful to you^